Assets/Scripts/EnemyBehaviour.cs

```diff
@@ -0,0 +1,197 @@
+/// \file Framework classes for Enemy actions and implementations thereof
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Enemy's level of involvement with the player
+/// </summary>
+public enum ENEMY_STATE
+{
+    /// <summary>
+    /// Enemy is unaware of player
+    /// </summary>
+    idle = 0,
+    /// <summary>
+    /// Enemy is aware of player but not engaged in conflict
```

Comment:
Just to try to keep the language consistent

```suggestion
    /// Enemy is aware of player but not engaged in combat
```